include/faker-cxx/image.h

```diff
@@ -34,14 +34,34 @@ enum class ImageCategory
  * @returns Random real image url from external service.
  *
  * @code
- * faker::image::imageUrl() // "https://loremflickr.com/640/480"
- * faker::image::imageUrl(800, 600) // "https://loremflickr.com/800/600"
- * faker::image::imageUrl(800, 600, ImageCategory::Animals) // "https://loremflickr.com/800/600/animals"
+ * faker::image::urlLoremFlickr() // "https://loremflickr.com/640/480"
+ * faker::image::urlLoremFlickr(800, 600) // "https://loremflickr.com/800/600"
+ * faker::image::urlLoremFlickr(800, 600, ImageCategory::Animals) // "https://loremflickr.com/800/600/animals"
  * @endcode
  */
-FAKER_CXX_EXPORT std::string imageUrl(unsigned width = 640, unsigned height = 480,
```

Comment:
please leave imageUrl as it was and inside it call either loremFlickr or Picsum function randomly